examples/src/main/java/kafka/examples/Consumer.java

```diff
@@ -145,6 +145,7 @@ public KafkaConsumer<Integer, String> createKafkaConsumer() {
         }
         // sets the reset offset policy in case of invalid or no offset
         props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+        props.put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, 1);
```

Comment:
Why should we add this?

Reply:
This is a left over from my test. Thanks for spotting.